catalog/spatial/kml/spatial-kml-transformer/src/main/resources/OSGI-INF/blueprint/blueprint.xml

```diff
@@ -72,4 +72,42 @@
         </cm:managed-component>
     </cm:managed-service-factory>
 
+    <bean id="kmlMetacardType" class="ddf.catalog.data.impl.MetacardTypeImpl">
+        <argument value="kml"/>
+        <argument>
+            <list>
+                <bean class="ddf.catalog.data.impl.types.AssociationsAttributes"/>
+                <bean class="ddf.catalog.data.impl.types.ContactAttributes"/>
+                <bean class="ddf.catalog.data.impl.types.CoreAttributes"/>
```

Comment:
This constructor already includes the CoreAttributes and this line can be removed

Reply:
Removed